planck-cljs/test/planck/io_test.cljs

```diff
@@ -27,6 +27,57 @@
     (is (= js/Date (type (:modified (planck.io/file-attributes "/tmp")))))
     (is (number? (:file-size (planck.io/file-attributes "/tmp"))))))
 
+(deftest predicates-test
+  (let [regular-file      "/tmp/plk-predicates-file.txt"
+        regular-directory "/tmp/plk-directory/"
+        hidden-file       "/tmp/.plk-predicates-hidden-file.txt"
+        hidden-directory "/tmp/.plk-hidden-directory/"]
+    (spit regular-file "Planck test case: regular file")
+    (io/make-parents regular-directory)
+    (spit hidden-file "Planck test case: hidden file")
+    (io/make-parents hidden-directory)
+    (testing "predicates"
+      (is (= false (planck.io/directory? nil)))
+      (is (= false (planck.io/exists? nil)))
+      (is (= false (planck.io/hidden-file? nil)))
+      (is (= false (planck.io/regular-file? nil)))
+      (is (= false (planck.io/symbolic-link? nil)))
```

Comment:
These tests would fail, triggering specs if instrumentation enabled.

Reply:
Should instrumentation be enabled when running the unit tests?

Reply:
https://github.com/planck-repl/planck/issues/866